snapshot/policy/scheduling_policy_test.go

```diff
@@ -173,6 +173,26 @@ func TestNextSnapshotTime(t *testing.T) {
 			wantTime:             time.Time{},
 			wantOK:               false,
 		},
+		{
+			pol: policy.SchedulingPolicy{
+				Cron: []string{"0 23 * * *"},
+			},
+			previousSnapshotTime: time.Date(2020, time.January, 1, 19, 0, 0, 0, time.Local),
```

Comment:
If I'm reading these cases correctly, now is before the previous snapshot, is that right?

Does it make sense to also have cases where now is after the previous snapshot? Both for cases where (a) now is before the time the next snapshot is supposed to be, and (b) now is after when the snapshot was supposed to have been taken.

What is the expected behavior for (b)? Would a new snapshot be taken right away?

Reply:
removed previousSnapshotTime since it was not essential here and confusing.